TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot_components/robot/Robot.java

```diff
@@ -65,6 +79,27 @@ public void toggleExtension(double distance) {
         }
     }
 
+
+    public void setExtenderPower(double power){
```

Comment:
fine for now since this needs improving later with limit switch and go to position, etc.